Resources/Prototypes/_White/Recipes/hidden_crafts.yml

```
@@ -84,4 +84,36 @@
         - !type:SpawnPrototype
           prototype: Igniter
         - !type:EmptyAllContainers
-        - !type:DeleteEntity
\ No newline at end of file
+        - !type:DeleteEntity
+
+- type: constructionGraph
```

yamllint

[warning] wrong indentation: expected 8 but found 0

(indentation)